src/args.py

```diff
@@ -60,4 +84,4 @@
 }
 
 # What thing do you have to add to make this work?
-f4(d)
\ No newline at end of file
+f4(d)
```

Comment:
you must add something here. what can you add to have d's values pass as kwargs?